build.gradle

```diff
@@ -148,16 +148,23 @@ task buildJniLib(type:Exec) {
     commandLine 'make'
 }
 
-test {
+task makeJniLib(type: Copy) {
     dependsOn buildJniLib
+    from "$buildDir/jni/release"
+    include "*"
```

Comment:
Minor: Should we specify to just copy the library? Making sure nothing else is copied

Reply:
Because the library extension is dependent on Operating System, I used wildcard here. Because only the library is placed in `jni/release` in the current build process, I believe this operation is okay.